src/python/pants/option/option_types.py

```diff
@@ -131,7 +129,7 @@ def __new__(
             user when running `help`.
         """
         self = super().__new__(cls)
-        self._flag_names = (flag_name, *additional_flag_names)
+        self._flag_names = (flag_name,)
```

Comment:
This can just be a singular `self._flag_name` field instead of a singleton tuple?

Reply:
We need it to be the tuple for the `LogLevelOption` to work. It overrides it to be a 2-tuple. 

This is an implementation detail though, hidden from end users.

Reply:
(Somewhat bespoke, but) It's easier to support the `-l` if we keep this a tuple because then the subclass can override.
Otherwise the subclass needs to override the callsit which is `__get__`, which involves more copy+pasting.

Alternatively we could add a new method `get_flag_names`, but that seems overkill.

(Just my 2c)